recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 mkdir build
-git checkout concat # temporary testing
+# git checkout concat # temporary testing
```

Comment:
Not sure if this is essential but hopefully the tarball is OK, if not we should issue a new release.